### PR TITLE
Added E1745 Motion Sensor

### PR DESCRIPTION
**Pull request by EpicLPer**
Added the E1745 Motion Sensor, an updated (and kind of worse) version of the E1525. Seems to be pretty similar tho with some differences on the PCB markings. Further investigation and firmware dumping has to be made still.

### CODE REVIEW

**Comment by sorryusernameisalreadytaken** [comment]
I think I have this device. Can I help to dump the firmware. I own a raspberry-pi 4.

File: PRODUCTS.md [hunk 1]
```diff
@@ -150,6 +150,35 @@ TRÅDFRI Module (unverified):
...
+
+* Battery: 2x CR2032
+
+TRÅDFRI Module (unverified):
+
+* Time selection button: Connected to tK3
+* Day/Night selection button: Connected to tK1
+* Pairing button: Connected to tK2
+
+* The motion detection dome is connected (one pin directly, one indirectly) to an "E93196B 0940A 60A" labelled 8-pin chip.
```
Comment (reedstrm) [inline PRODUCTS.md:165]: This chip appears to be an Elmos E931.96 low power PIR sensor controller - https://www.elmos.com/english/products/sensor-ics/passive-infrared/pyroelectric-sensor-signal-processor-ssp/e93196.html

**Comment by kollokollo** [comment]
It looks like there is a new version of the motion sensor available: The difference is that there is no button for time selection, instead there is a button for brightness (30% or 100%). The time is always about 3 Minutes.

**Comment by basilfx** [comment]
I have added the images to the repository and cherry-picked this commit.

**Pull request closed without merging**